### DISCUSSION TITLE
Best way to share the .kdbx database

### DISCUSSION THREAD

**Discussion by aroldobossoni** [keepassxreboot/keepassxc#6065] [Q&A] [answered]
- I know that keepassxc does not directly connect to cloud or ftp to load the .kdbx database.
- I know that keepassxc is able to reload the database when modified externally.
- I use keepassxc on 3 different operating systems simultaneously, Ubuntu, Android and Windows 10. However, there have been errors such as bad files, the file synchronization application changes the file name and loses new changes.

In your experience what is the best way to share the .kdbx database between those same platforms, ftp, dropbox, google drive, SSH?

**Comment by phoerious** [comment, marked as answer]
Dropbox, Google Drive, OneDrive, Nextcloud. Any of those work perfectly. But use the original sync client, not some half-baked GVfs Nautilus integration.
